3ds2playground/src/main/kotlin/com/stripe/android/stripe3ds2playground/App.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.stripe3ds2playground
+
+import android.app.Application
+import android.os.StrictMode
+
+class App : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        StrictMode.setThreadPolicy(
+            StrictMode.ThreadPolicy.Builder()
+                .detectDiskReads()
+                .detectDiskWrites()
+                .detectAll()
+                .penaltyLog()
+                // penaltyDeath() causes tests to fail in CI
```

Comment:
can you delete the commented out lines here?

Reply:
Done 👍